src/test/java/edu/harvard/iq/dataverse/bannersandmessages/validation/BannerErrorHandlerTest.java

```diff
@@ -41,6 +41,7 @@
     private ArgumentCaptor<FacesMessage> facesMesssage;
 
     private static final Path BANNER_PATH = Paths.get("src/test/resources/images/banner.png");
+    private static final Path TXT_PATH = Paths.get("src/test/resources/images/sample.txt");
```

Comment:
You are depending here on the fact that current directory is main repository directory which is not always the case.
These files should be rather taken from classpath.